src/dom/dom.js

```diff
@@ -2011,7 +2006,7 @@ p5.Element.prototype.hide = function() {
  * <div class='norender'><code>
  * let div = createDiv('this is a div');
  * div.size(100, 100);
- * let img = createImg('assets/laDefense.jpg', () => {
+ * let img = createImg('assets/laDefense.jpg', 'a test image', '', () => {
```

Comment:
can you make this alt text more descriptive of the image content itself to serve as a good example for users?

Reply:
oh good catch. so i looked at the image and literally couldn't figure out what i was looking at 😞, so i used the test image text. i just looked up "la defense" and its a district in paris, so maybe this is a cropped photo of some buildings there? @stalgiag do you have any idea? happy to make this text better, just don't know what this image is...

Reply:
lol I couldn't say. in this case, since it's an example of alt text, let's swap the image to something more recognizable to provide a better example of text describing an image

Reply:
that makes so much sense lol

Reply:
I tried 'search by image' out of curiosity and couldn't find another example of that asset on the internet. @lmccart [mysteriously added it 5 years ago](https://github.com/processing/p5.js/commits/master/docs/yuidoc-p5-theme/assets/laDefense.jpg), and now is suggesting replacing it. I don't like to throw the word conspiracy around but ...

Reply:
woah there if people are whispering conspiracy @REAS must’ve had something to do with it [..](https://github.com/processing/processing-docs/issues/605)

Reply:
Uhhhh, it's La Grande Arche de la Défense: https://en.wikipedia.org/wiki/Grande_Arche. Maybe should be a kitten instead?

Reply:
ohhhhhh! wow i would never have figured that out. i thought about deleting the image from the repo to keep the conspiracy going, but we use it in many other examples.

![animated gif of a pyramid and an eye above it a lá the illuminati or freemasons (or some other secret society)](https://media.giphy.com/media/LX3Q9vcjwhKYE/giphy.gif)

for this instance, i've changed it to a wonderful photo of a mountain in https://github.com/processing/p5.js/pull/3890/commits/c5aeea79c81af6c4aaf640a3a3b98b62500ef3f4! yay!